jre_emul/Classes/com/google/j2objc/security/IosRSAKey.java

```diff
@@ -41,6 +41,10 @@
   static final String PUBLIC_KEY_TAG = "com.google.j2objc.security.publickey";
   static final String PRIVATE_KEY_TAG = "com.google.j2objc.security.privatekey";
   private static final long serialVersionUID = 1L;
+  
+  public IosRSAKey() { 
```

Comment:
Please remove "public" (our mistake, because the other constructors in this class should be package-private, too).